picoquic/util.c

```diff
@@ -667,8 +667,10 @@ uint8_t* picoquic_frames_l_v_encode(uint8_t* bytes, const uint8_t* bytes_max, si
 {
     if ((bytes = picoquic_frames_varlen_encode(bytes, bytes_max, l)) != NULL &&
         (bytes + l) <= bytes_max) {
-        memcpy(bytes, v, l);
-        bytes += l;
+        if (l > 0) {
```

Comment:
I think this check is a bit superfluous since memcpy can handle copying 0 bytes.

Reply:
OK.